### PR TITLE
Configure m-compiler-p to use release flag for earlier JDKs

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

**Comment by pzygielo** [comment]
For first workflow run failure:
- mojohaus/mojo-parent#326

File: pom.xml [hunk 1]
```diff
@@ -215,9 +215,9 @@
     </profile>
 
     <profile>
-      <id>java12</id>
+      <id>java9+</id>
       <activation>
-        <jdk>[12,)</jdk>
+        <jdk>[9,)</jdk>
```
Comment (slawekjaranowski) [inline pom.xml:220]: IMHO it can be even in parent pom

Comment (pzygielo) [reply pom.xml:220]: > IMHO it can be even in parent pom

Agree. Once it is - this should be dropped.

**Comment by pzygielo** [comment]
May I ask for review, please?
If there is parent with such profile - please upgrade to it.
It there isn't - please request changes or accept and merge or close this PR.

**Comment by pzygielo** [comment]
Thanks for checking!

**Pull request merged**